test/JDBC/input/BABEL-SP_TABLES.sql

```diff
@@ -103,6 +109,42 @@ go
 exec sp_tables @table_name = 't_s[o-p]tables2'
 go
 
+exec sp_tables @table_name = 't[_]sptables', @table_type = "'TABLE'"
```

Comment:
Thought: Should we add testcases for other wildcards supported by TSQL, if any?

Reply:
this test file already have tests for different wildcards supported by TSQL, if we are taking about specifically for square bracket wild card then the test I had added and existing tests should be sufficient to cover all the scenario's.

Reply:
ack, thanks for confirming it.